File: ktpm-front-angular/src/model/Bill.ts
import { Guid } from "guid-typescript";

export class Bill {
  public id: number;
  public clientId: number;
  public tableOrderId: number;
  public totalPrice: number;
  public dateOrder: Date;
  public statusPay: any;
  constructor(id: number, clientId: number, tableOrderId: number, totalPrice: number, dateOrder: Date, statusPay: any) {
    this.id = id;
    this.clientId = clientId;
    this.tableOrderId = tableOrderId;
    this.totalPrice = totalPrice;
    this.dateOrder = dateOrder;
    this.statusPay = statusPay;
  }
}

File: KTPM/src/main/java/com/example/ktpm/model/Bill.java
package com.example.ktpm.model;


import jakarta.persistence.*;
import lombok.Data;

import java.util.Date;
import java.util.List;

@Entity
@Data
public class Bill {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    @Column(name = "clientId")
    private Long clientId;
    @Column(name = "tableOrderId")

    private Long tableOrderId;
    @Column(name = "totalPrice")
    private Long totalPrice;
    @Column(name = "dateOrder")
    private Date dateOrder;
    @Column(name = "statusPay")
    private String statusPay;

    public String getStatusPay() {
        return statusPay;
    }

    public void setStatusPay(String statusPay) {
        this.statusPay = statusPay;
    }

    public Long getId() {
        return id;
    }
    public void setId(Long id) {
        this.id = id;
    }

    public Long getClientId() {
        return clientId;
    }

    public void setClientId(Long clientId) {
        this.clientId = clientId;
    }

    public Long getTableOrderId() {
        return tableOrderId;
    }

    public void setTableOrderId(Long tableId) {
        this.tableOrderId = tableId;
    }

    public Long getTotalPrice() {
        return totalPrice;
    }

    public void setTotalPrice(Long totalPrice) {
        this.totalPrice = totalPrice;
    }

    public Date getDateOrder() {
        return dateOrder;
    }

    public void setDateOrder(Date dateOrder) {
        this.dateOrder = dateOrder;
    }
}
